src/fp.rs

```diff
@@ -312,6 +312,11 @@ impl Fp {
         Fp(v)
     }
 
+    /// Internal representation of `Fp`
+    pub const fn internal_repr(&self) -> &[u64; 6] {
+        &self.0
+    }
```

Comment:
Not to solve in this PR, but this method always bothered me; we should go for a more idiomatic rust naming convention. What do you think @vlopes11 ?

Reply:
As discussed, will create an issue for that